components/brave_wallet/browser/eth_data_builder.cc

```diff
@@ -125,6 +145,13 @@ bool ContentHash(const std::string& domain, std::string* data) {
   return ConcatHexStrings(hex_strings, data);
 }
 
+bool Addr(const std::string& domain, std::string* data) {
```

Comment:
You could add a test for this one and `Get` pretty easily in `components/brave_wallet/browser/eth_data_builder_unittest.cc`